CHANGES.rst

```diff
@@ -8,7 +8,13 @@ Unreleased
 - Side-Navigation: Adjusted size sligthly, added hover-effect on selected item,
   added border on active item and fixed monitoring icon
 
+<<<<<<< HEAD
 - Added ability to correctly display nested arrays in OBJECT(IGNORED) columns
+=======
+- Move Monitoring to Overview and remove dedicated section. Monitoring now 
+  always show last 15 minutes of QPS and query duration in 10 sec intervals.
+  Slighty adjusted appearance of graphs.
+>>>>>>> 504871e (Move query monitoring to overview and remove)
```

Comment:
It looks like conflict markers slipped in here.

Reply:
Thanks ... VS Code played tricks on me 😬